### ISSUE TITLE
Improve contrast ratio

### ISSUE THREAD

**Issue by ajanec01** [craftcms/cms#7289]
### Little contrast ratio for focus indication and text

The red login button changes contrast ratio to indicate keyboard focus.(#E12D39 to #d61f2b). It's currently only 1.13. 

Contrast ratio for the colours that change on the input fields is 2.05:1.

The craft cms link (svg) has contrast ratio of 1.49:1. It also has default browser focus indication which is likely become a failure in the [drafted focus appearance criterion.](https://www.w3.org/TR/WCAG22/#focus-appearance-minimum) 

### Additional info

- Craft version: 3.5.17.1

**Comment by missmatsuko** [comment]
We will add more prominent focus styles to fix this issue and #6886

**Comment by missmatsuko** [comment]
This should be resolved with #7276 and #7680 🙂
